### DISCUSSION TITLE
Transfer audio files to IOS

### DISCUSSION THREAD

**Discussion by ggrinton** [localsend/localsend#1140] [Discussion (English, International)]
Can I use LocalSend to transfer audio files to (or from) IOS? I tried, and it appeared to work, but I cannot find anywhere that the file might have been stored on IOS after the transfer.

Thanks.

**Comment by lcopley** [comment]
No answer, but same problem.  Maybe adding more voices can help.

**Comment by mice2100** [comment]
I find it with files, under "On My iPhone"/Downloads
